src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -243,6 +243,22 @@ impl<'a> Resolver<'a> {
                 }
 
                 for &(ref tree, id) in items {
+                    // Ensure use keywords or absolute imports aren't used in a nested list
+                    let subprefix = &tree.prefix.segments;
```

Comment:
Looks like this commit duplicates https://github.com/rust-lang/rust/pull/45771/commits/90f5cfdfbd3c9e39f897fd2271c7b1c7fdaae58e, but only for import paths.